packages/cli/src/lib/bundler/types.ts

```diff
@@ -25,6 +25,8 @@ export type BundlingOptions = {
   baseUrl: URL;
 };
 
+export type BackendBundlingOptions = Omit<BundlingOptions, 'baseUrl'>;
```

Comment:
Meh, type it out instead imo :p I think you actually only need the first 2 fields from BundlingOptions?